tests/Silverback.Integration.Tests.E2E/Kafka/BasicTests.cs

```diff
@@ -1019,5 +1020,48 @@ public async Task Inbound_FromMultiplePartitionsWithLimitedParallelism_Concurren
 
             receivedMessages.Should().HaveCount(12);
         }
+
+        [Fact]
+        public async Task ConsumerAssing_GroupIdNotSet_OffsetsNotCommitted()
+        {
+            int receivedMessages = 0;
+            Host.ConfigureServices(
+                    services => services
+                        .AddLogging()
+                        .AddSilverback()
+                        .UseModel()
+                        .WithConnectionToMessageBroker(options => options.AddMockedKafka())
+                        .AddKafkaEndpoints(
+                            endpoints => endpoints
+                                .Configure(
+                                    config =>
+                                    {
+                                        config.BootstrapServers = "PLAINTEXT://e2e";
+                                    })
+                                .AddOutbound<IIntegrationEvent>(
+                                    endpoint => endpoint.ProduceTo(DefaultTopicName))
+                                .AddInbound(
+                                    endpoint => endpoint
+                                        .ConsumeFrom(DefaultTopicName, topics => topics)
+                                        .Configure(
+                                            config =>
+                                            {
+                                                config.GroupId = string.Empty; // No group.id is set
```

Comment:
The most natural way to do this is to omit the assignment. Would that work the same? It's maybe the happy case we should test.

Reply:
You're right, thanks!
I added a new unit test: https://github.com/BEagle1984/silverback/pull/172/commits/3fe86322dc9c99d9771c6009985324ab784b397d